package.json

```diff
@@ -110,11 +99,17 @@
 	},
 	"dependencies": {
 		"@woocommerce/components": "4.0.0",
+		"@wordpress/blocks": "6.9.0",
 		"@wordpress/components": "8.5.0",
+		"@wordpress/data-controls": "1.5.0",
+		"@wordpress/editor": "9.9.0",
+		"@wordpress/element": "2.10.0",
 		"@wordpress/html-entities": "2.5.0",
+		"@wordpress/i18n": "3.7.0",
+		"@wordpress/is-shallow-equal": "1.6.1",
```

Comment:
I think the _only_ `@wordpress` package that should be in this list is `@wordpress/components` because all the others are used in the production build as an external (built bundle from WordPress core). So the other imports aren't _required_ for production builds (we just pull them in for running tests which use them).  `devDependencies` is a better fit in that case.

We'd keep `@wordpress/components` in this list because although we use it as an external, we also use it directly via the `wordpress-components` alias pulling from what is in `node_modules`.

From the [npm documentation](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file):

> "dependencies": Packages required by your application in production.
> "devDependencies": Packages that are only needed for local development and testing.

Reply:
> I think the _only_ `@wordpress` package that should be in this list is `@wordpress/components` because all the others are used in the production build as an external (built bundle from WordPress core).

Good point, I wasn't sure whether externals belonged to `dependencies` or `devDependencies`. Changed in b23dd357ba947a640b58beb6195fb499a0daa902.

> We'd keep `@wordpress/components` in this list because although we use it as an external, we also use it directly via the `wordpress-components` alias pulling from what is in `node_modules`.

We already have the entry
```
"wordpress-components": "npm:@wordpress/components@8.5.0"
```
which should account for that, right? I moved `@wordpress/components` into `devDependencies` too and kept `wordpress-components`  in `dependencies`. Does that make sense?